src/main/java/org/apache/mesos/hdfs/Scheduler.java

```diff
@@ -177,6 +142,46 @@ private void launchNode(SchedulerDriver driver, Offer offer,
     driver.launchTasks(Arrays.asList(offer.getId()), tasks);
   }
     
+  private ExecutorInfo createExecutor(String taskIdName, String nodeName, String executorName) {
+    int confServerPort = conf.getConfigServerPort();
+    List<Resource> resources = getExecutorResources();
+    ExecutorInfo executorInfo = ExecutorInfo.newBuilder()
+        .setName(nodeName + " executor")
+        .setExecutorId(ExecutorID.newBuilder().setValue("executor." + taskIdName).build())
+        .addAllResources(resources)
+        .setCommand(CommandInfo.newBuilder()
+        .addAllUris(Arrays.asList(
+            CommandInfo.URI.newBuilder().setValue(
+                conf.getExecUri())
+                .build(),
+            CommandInfo.URI.newBuilder().setValue(
+                String.format("http://%s:%d/hdfs-site.xml", localhost, confServerPort))
```

Comment:
good point to make note of. We will have to consider both these cases if we have IPv6 users since the format is different.